pmlc/dialect/stdx/ir/ops.td

```diff
@@ -43,4 +44,17 @@ def FPToUIOp :
   }]>];
 }
 
+class StdX_Plain_Op<string mnemonic, list<OpTrait> traits = []> :
+    Op<StdX_Dialect, mnemonic, traits> {
+}
```

Comment:
Why do we need a different base class?

Reply:
I wanted to use assemblyFormat and therefore do not want the declarations for custom print/parse/verify functions supplied in StdX_Op.  If I define ReshapeOp based on StdX_Op instead, the compiler complains that I have not provided implementations.